_i18n/it/technical-specs.md

```diff
@@ -78,14 +78,14 @@
                     </div>
 
 <div markdown="1">
-* primo, curva principale: ~18.132 milioni di monete entro la fine di Maggio 2022
-* poi, curva di emissione: 0.6 XMR per blocco, entra in vigore appena l'emissione principale è finita, il che si traduce in  <1% inflazione decrescente nel tempo
+* prima fase, curva principale: ~18.132 milioni di monete entro la fine di Maggio 2022
+* seconda fase, curva di emissione: 0.6 XMR per blocco (se generato ogni 2 minuti), subentra alla prima fase quando l'emissione principale è terminata; questo si traduce in un tasso di inflazione <1% e decrescente nel tempo
 * vedi [grafici e dettagli](https://www.reddit.com/r/Monero/comments/512kwh/useful_for_learning_about_monero_coin_emission/)
 </div>
 
                     <div class="row">
                         <div class="col">
-                            <h3>Fornitura massima</h3>
+                            <h3>Massima offerta di moneta</h3>
```

Comment:
I like more the past string. `massima offerta di moneta` doesn't sound correct to me

Reply:
You're right, I checked the term "fornitura massima" and it is used as a technical term. Sorry.
Agreed the revert,